__tests__/endpoints.test.js

```diff
@@ -14,12 +14,39 @@ describe("/api/topics", () => {
       .expect(200)
       .then((response) => {
         expect(response.body.topics).toEqual(expect.any(Array));
-        if (response.body.topics.length > 0) {
-          for (let i = 0; i < response.body.topics.length; i++) {
-            expect(Object.keys(response.body.topics[i])).toEqual(
-              expect.arrayContaining(["slug", "description"])
-            );
-          }
+        expect(response.body.topics).not.toHaveLength(0);
+        for (let i = 0; i < response.body.topics.length; i++) {
```

Comment:
Similarly to comment below - consider using forEach rather than a for loop, it just reads a bit nicer :)